jenkins-agent/acr-agents-build-task.yaml

```
@@ -0,0 +1,23 @@
+version: v1.1.0
+
+steps:
+  - id: build-minimal-agent
+    build: -f minimal.Dockerfile -t {{.Run.Registry}}/jenkins/minimal-agent:{{.Run.ID}} -t {{.Run.Registry}}/jenkins/minimal-agent:latest .
```

this should be your id in the container registry

```suggestion
    build: -f minimal.Dockerfile -t {{.Run.Registry}}/gatling:{{.Run.ID}} -t {{.Run.Registry}}/jenkins/minimal-agent:latest .
```